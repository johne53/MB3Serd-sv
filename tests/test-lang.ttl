@prefix : <http://example.org/test-lang#> .

:thing :greeting "Hello"@en ;
       :greeting "Howdy"@en-us ;
       :greeting "Bonjour"@fr .
