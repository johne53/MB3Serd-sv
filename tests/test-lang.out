<http://example.org/test-lang#thing> <http://example.org/test-lang#greeting> "Hello"@en .
<http://example.org/test-lang#thing> <http://example.org/test-lang#greeting> "Howdy"@en-us .
<http://example.org/test-lang#thing> <http://example.org/test-lang#greeting> "Bonjour"@fr .
